bundle/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -255,20 +256,30 @@ spec:
           - update
           - watch
         - apiGroups:
-          - build.openshift.io
+          - ""
           resources:
-          - '*'
+          - configmaps
+          - events
+          - pods
+          - secrets
+          - serviceaccounts
+          - services
           verbs:
           - create
           - delete
           - get
           - list
+          - patch
           - update
           - watch
         - apiGroups:
           - ""
           resources:
-          - '*'
+          - configmaps
```

Comment:
This group is same as above, the only difference is `serviceaccounts` resource, could they be merged to one?